pkg/shell/cockpit-dashboard.js

```diff
@@ -471,6 +471,11 @@ PageDashboard.prototype = {
                 });
 
                 target.html(text);
+                $(".delete-" + cockpit.transport.host).tooltip({
```

Comment:
Because other pages (ie: not the dashboard(s)) will be shown for other hosts. The dashboard is only ever shown when 'cockpit.transport.host' is 'localhost'